lib/fake_ifdh.py

```diff
@@ -41,6 +41,20 @@
 DEFAULT_ROLE = "Analysis"
 
 
+def init_scitokens() -> None:
```

Comment:
Would you mind putting a docstring in there that explains 1) that we're doing this to NFS-proof the key-caching, and 2) That we're dynamically generating a config-file that the scitokens library will use?  I just want to make sure that six-months-from-now-us will know what this is even for.